packages/crml-attestation/package.json

```diff
@@ -15,10 +15,10 @@
     "dependencies": {
         "@cennznet/api": "^0.20.7",
```

Comment:
Do we want these to be `^1.0.0-alpha.0`?

Reply:
Yes, I changed `0.20.7` to be `1.0.0-alpha.0` manually in a following commit, and I am trying to test it locally.

Reply:
Thanks Ben!